articles/TOC.yml

```diff
@@ -1,44 +1,52 @@
 - name: Microsoft Quantum Documentation
   href: index.yml
-- name: Overview
+- name: Getting started
   items:
-  - name: Get started
+  - name: What is quantum computing?
+    href: overview/what-is-quantum-computing.md
+  - name: What can a quantum computer do?
+    href: overview/quantum-computers.md
+  - name: Why learn quantum computing?
+    href: overview/why-learn-quantum-computing.md
+  - name: What are Q# and QDK?
+    href: overview/what-is-qsharp.md
+  - name: How can I learn more?
+    href: overview/how-to-learn-quantum-computing.md
+- name: The QDK introduction
+  items:
+  - name: Get started with the Quantum Development Kit
     href: welcome.md
-  - name: Five questions about quantum computing
+  - name: Install and update
     items:
-    - name: What is quantum computing?
-      href: overview/what-is-quantum-computing.md
-    - name: What can a quantum computer do?
-      href: overview/quantum-computers.md
-    - name: Why learn quantum computing?
-      href: overview/why-learn-quantum-computing.md
-    - name: What is Q#?
-      href: overview/what-is-qsharp.md
-    - name: How to learn quantum computing?
-      href: overview/how-to-learn-quantum-computing.md
-- name: Install and update
+    - name: Install the QDK
+      items:
+      - name: Installation overview
+        href: install-guide/index.md
+      - name: Q# + C#
+        href: install-guide/csinstall.md
+      - name: Q# + Python
+        href: install-guide/pyinstall.md
+      - name: Q# Jupyter notebooks
+        href: install-guide/qjupyter.md
+    - name: Update the QDK
+      href: install-guide/update.md
+    - name: How to create a Q# project
+      href: how-to-guides/create-qsharp-project.md
+- name: Q# examples
```

Comment:
I think this is a huge improvement over what we had, however, I don't know if "Q# examples" is the correct title since the Katas and the Quickstarts are not really examples. I think the previous title was ok ("Q# learning resources"), only that the material was sparse across different sections ('How-to guides', 'Quickstarts'...).

Reply:
Will change to "Q# learning resources"